docs/guides/index.markdown

```diff
@@ -3,22 +3,36 @@ layout: basic
 title: Guides
 ---
 
-### Guides
+## Guides
 
-#### User Guides
+### User Guides
 
 * [AeroGear Android User Guide](aerogear-android)
 * [AeroGear iOS 1.X User Guide](aerogear-ios)
 * [AeroGear iOS 2.X User Guide](aerogear-ios-2.X)
-* [AeroGear Two-Factor Authentication with OTP](AeroGear-OTP)
-* [AeroGear Two-Factor Authentication with OTP as Cordova Plugin](aerogear-cordova/AerogearCordovaOTP)
 
-#### UnifiedPush Server
+### UnifiedPush Server
 
 * [AeroGear UnifiedPush Server resources](../unifiedpush)
 
+### Security
 
-#### Contributing to AeroGear
+* OTP
+  * [AeroGear Two-Factor Authentication with OTP](AeroGear-OTP)
+  * [AeroGear Two-Factor Authentication with OTP as Cordova Plugin](aerogear-cordova/AerogearCordovaOTP)
+
+* OAuth2
+  1. [Basic steps](security/oauth2-basic-steps)
+  2. Client libraries
+
+     The following SDKs were crafted with the simplicity in mind and are constantly improving at each release.
+
+     * [Android client library](http://search.maven.org/#artifactdetails%7Corg.jboss.aerogear%7Caerogear-android-authz%7C2.0.0%7Caar)
+     * [iOS client library](http://cocoapods.org/?q=aerogearoauth2)
+     * [Apache Cordova client library](https://github.com/aerogear/aerogear-oauth2-cordova/releases/tag/v1.0.2)
```

Comment:
Let's use this link: http://plugins.cordova.io/#/package/org.jboss.aerogear.cordova.oauth2